build-status.el

```diff
@@ -375,8 +375,7 @@ Signals an error if the response does not contain an HTTP 200 status code."
 ;;;###autoload
 (define-minor-mode build-status-mode
   "Monitor the build status of the buffer's project."
-  :global t
```

Comment:
Because in some buffer files which are not in CI project will report error with `(error "Not in CI project")`. I use a custom mode-line, but reporting error will cause my custom mode-line failed.
Maybe should change `(error "Not in CI project")`.